src/Arcus.Messaging.Pumps.ServiceBus/AzureServiceBusMessagePump.cs

```diff
@@ -661,52 +540,58 @@ protected override Task PreProcessMessageAsync<TMessageContext>(MessageHandler m
             return Task.CompletedTask;
         }
 
-        private async Task ProcessMessageWithFallbackAsync(Message message, CancellationToken cancellationToken, MessageCorrelationInfo correlationInfo)
+        // TODO: remove 'old' workings after the background jobs package is updated with the new messaging package.
+        private async Task ProcessMessageWithFallbackAsync(
+            ProcessMessageEventArgs args, 
+            AzureServiceBusMessageContext messageContext, 
+            MessageCorrelationInfo correlationInfo, 
+            CancellationToken cancellationToken)
         {
             try
             {
-                Logger.LogTrace("Received message '{MessageId}'", message.MessageId);
+                Logger.LogTrace("Received message '{MessageId}'", args.Message.MessageId);
```

Comment:
What if message is null?

Reply:
Is already checked in a the parent method of this one, but can be added of course.